src/lightning_lite/connector.py

```diff
@@ -523,6 +531,27 @@ def _lazy_init_strategy(self) -> None:
                 f" found {self.strategy.__class__.__name__}."
             )
 
+    @staticmethod
```

Comment:
Is this static so it can be overridden? Move outside the class instead? Maybe even define in the `cli.py` file so the definition and parsing is together

Reply:
It belongs to this class, because only this class has custom logic in the init that corresponds to the implementation in this function.